FinniversKit/Sources/Components/Switch/SwitchView.swift

```diff
@@ -39,7 +39,7 @@ public class SwitchView: UIView {
 
     private lazy var uiSwitch: UISwitch = {
         let uiSwitch = UISwitch(withAutoLayout: true)
-        uiSwitch.onTintColor = .btnPrimary //DARK
+        uiSwitch.onTintColor = .nmpBrandControlSelected //DARK
```

Comment:
Can we remove unneeded comment here?

Reply:
haha sure, not sure what this comment means 🤷 I'll delete it.